cli_tools_e2e_test/gce_image_import_export/test_suites/import/import_tests.go

```diff
@@ -154,7 +155,18 @@ var basicCases = []*testCase{
 		source:   "projects/compute-image-tools-test/global/images/rhel-8-2",
 		os:       "rhel-8",
 		inspect:  true,
-	}, {
+	},
+
+	// EL - Error cases
+	{
+		caseName: "el-allow-extra-dirs-in-lib-modules", // b/168774581
+		source:   "projects/compute-image-tools-test/global/images/el-depmod-lib-modules",
```

Comment:
minor: this image could have been named e.g. el-depmod-extra-lib-modules to indicate there are more lib modules than just the ones representing the kernel

Reply:
done